apps/src/codebridge/Console/index.tsx

```diff
@@ -54,11 +58,27 @@ const Console: React.FunctionComponent = () => {
         {codeOutput.map((outputLine, index) => {
           if (outputLine.type === 'img') {
             return (
-              <img
-                key={index}
-                src={`data:image/png;base64,${outputLine.contents}`}
-                alt="matplotlib_image"
-              />
+              <div key={index}>
+                <img
+                  src={`data:image/png;base64,${outputLine.contents}`}
+                  alt="matplotlib_image"
+                />
+                <Button
+                  color={buttonColors.black}
+                  disabled={false}
+                  icon={{iconName: 'up-right-from-square', iconStyle: 'solid'}}
+                  isIconOnly={true}
+                  onClick={() => setGraphModalOpen(true)}
+                  size="xs"
+                  type="primary"
+                />
+                {graphModalOpen && (
```

Comment:
is it possible to have more than one modal open?

Reply:
No, because there's no way to open a second without closing the first given that you have to click dismiss or click outside the first modal to access the "open in new window" icon of the second which will close the first.

Reply:
I can imagine a future world where it might be nice to have resizable & draggable popups that don't darken the background so a student could compare graphs side by side; but that felt like a bridge we could cross if/when we come to it.